src/components/ImageAlignmentPage.vue

```diff
@@ -365,6 +380,56 @@
            });
      },
 
+     deleteOpticalImg() {
```

Comment:
As I wrote in the review of changes in GQL, I'd move all the deletion process to `sm-engine`

Reply:
I just had a small discussion with @intsco and I think I will try to handle all deletion steps on the GQL side in one method